js/data.js

```diff
@@ -38,24 +38,32 @@ const messages = [
 
 const names = ['Федя', 'Яна', 'Николай', 'Ксения', 'Иван', 'Слава'];
 
+const comments = [];
+for (let i = 1; i <= 15; i++) {
+  comments.push(makeComments(i));
+}
+
 const photos = [];
 for (let i = 1; i <= 25; i++) {
   photos.push(makePhotoDescription(i));
 }
 
+function makeComments(id) {
+  return {
+    id: id,
+    avatar: `img/avatar-${getRandomInteger(1, 6)}.svg`,
+    message: getRandomString(messages),
+    name: getRandomString(names),
+  };
+}
+
 function makePhotoDescription(id) {
   return {
     id: id,
     url: `photos/${id}.jpg`,
     description: getRandomString(descriptions),
     likes: getRandomInteger(15, 200),
-    comments: {
-      id: id,
-      avatar: `img/avatar-${getRandomInteger(1, 6)}.svg`,
-      message: getRandomString(messages),
-      name: getRandomString(names),
-    },
   };
 }
```

Comment:
у тебя же принуипиально ничего не поменялось, ты всё равно выводишь для всех фотографий одни и теже комменты, тебе нужно оставить comments внутри каждой фотки, и желательно что бы они там у разных карточек были разные, то есть что то типа 
```suggestion
return {
    id: id,
    url: `photos/${id}.jpg`,
    description: getRandomString(descriptions),
    likes: getRandomInteger(15, 200),
    comments: Array.from({length: getRandomInteger(5, 25)}, (v, k) => makeComments(k+1));
  };
```